Gemfile

```diff
@@ -45,6 +45,8 @@ group :development, :test do
   gem 'rubocop'
   gem 'dotenv-rails'
 
+  gem 'factory_girl_rails', '~> 4.4.1'
```

Comment:
Is there a reason why factory_girl is needed in the development environment in addition to test?